person.rb

```diff
@@ -3,7 +3,7 @@
 require './trimmer_decorator'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] `@rentals` should be a `Getter and Setter` which means an `attr_accessor`, please kindly fix that 🙏🏾 
- [ ] Also you should have a method for adding rentals and the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
It can be similar to 👇🏾 

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```